# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsa-tools/glibc.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- a/envy24control/strstr_icase_blank.c
+++ b/envy24control/strstr_icase_blank.c
@@ -48,7 +48,6 @@ int strstr_icase_blank(const char * const string1, const char * const string2)
 	char search_string[MAX_SEARCH_FIELD_LENGTH];
 	char *pstr;
 	int pos_first_non_blank;
-	int isblank(int c);

 	strncpy(search_string, string2, MAX_SEARCH_FIELD_LENGTH);
 	search_string[MAX_SEARCH_FIELD_LENGTH - 1] = '\0';

